plugins/kinet/README.md

```diff
@@ -2,12 +2,34 @@ KiNET Plugin
 ============
 
 This plugin creates a single device with multiple output ports. Each port
-represents a power supply. This plugin uses the V1 DMX-Out version of the
-KiNET protocol.
+represents a power supply. This plugin supports both the DMX Out (V1) and the
+Port Out (V2) modes of the KiNET protocol.
 
 
 ## Config file: `ola-kinet.conf`
 
-`power_supply = <ip>`  
+### Per Power Supply Settings
+
+`power_supply = <ip>`
 The IP of the power supply to send to. You can communicate with more than
-one power supply by adding multiple `power_supply =` lines
+one power supply by adding multiple `power_supply =` lines.
+
+`<ip>-mode = [dmxout|portout]`
+The mode of KiNET to send to the power supply. DMX Out is sometimes known as V1
+and Port Out as V2.
+
+In general, it is highly recommended to use the Port Out mode if your power
+supply supports it. Note that some newer devices only support the Port Out
+protocol. On some devices it is also necessary to use the Port Out protocol in
```

Comment:
```suggestion
In general, it is highly recommended to use the Port Out mode if your power
supply supports it, we default to DMX Out mode for backwards compatibility reasons. Note that some newer devices only support the Port Out
protocol. On some devices it is also necessary to use the Port Out protocol in
```

> Note that some newer devices only support the Port Out protocol.

As an end user, how can I tell? In QPP, or do I jut have to RTFM?

Reply:
RTFM (if it is CK they may not even put it there), or to be honest just trial and error. Considering the oldest device I have used from 2005 supports Port Out, it shouldn't be a surprise really. Though if the device supports more channels than can fit in one universe (like the sPDS-480ca), then it is very likely.

Reply:
Added backwards compatibility note.

Reply:
> RTFM (if it is CK they may not even put it there), or to be honest just trial and error. Considering the oldest device I have used from 2005 supports Port Out, it shouldn't be a surprise really. Though if the device supports more channels than can fit in one universe (like the sPDS-480ca), then it is very likely.

I wonder if we should be more positive/pro-active with a try it and see note, although I guess it's only relevant if its driving more than a universe in which case it presumably probably does anyway.

Reply:
Yeah it does already say that devices that require more channels will need it. I can add a note that Port Out should be tried first.

Reply:
Added note to try it first. There are already notes about the newer devices and the larger-than-one-universe.